### ISSUE TITLE
Error message on setup: 'A later version of Brackets is already installed. Setup will now exit'

### ISSUE THREAD

**Issue by mobesa** [adobe/brackets#10173]
Hey everyone,

I was trying to install Brackets 1.0 + Adobe Extract (downloaded the .msi from brackets.io on windows 7 x64 via Chrome) when I got this message:
"A later version of Brackets is already installed. Setup will now exit."

Help?

P.S.: I hadn't installed Brackets before (duh).

**Comment by JeffryBooher** [comment]
@mobesa could you help us out with trying to figure out the problem? I know you have it installed by the zip but we're thinking there is something with the Windows Installer database that is causing this. Can you try the following:

Open command prompt
cd to folder containing the installer MSI file.
Run msiexec /i Brackets.1.0.Extract.msi /l*v brackets_log.txt

and paste the contents of the log file to this thread?
Thanks!

**Comment by mobesa** [comment]
@JeffryBooher sorry for being so late...
I did as you said and here's the log that I got:
```
=== Verbose logging started: 15/03/36  22:17:00  Build type: SHIP UNICODE 5.00.7601.00  Calling process: C:\Windows\system32\msiexec.exe ===
MSI (c) (AC:74) [22:17:00:221]: Font created.  Charset: Req=178, Ret=178, Font: Req=MS Shell Dlg, Ret=MS Shell Dlg

MSI (c) (AC:74) [22:17:00:221]: Font created.  Charset: Req=178, Ret=178, Font: Req=MS Shell Dlg, Ret=MS Shell Dlg

MSI (c) (AC:9C) [22:17:00:308]: Resetting cached policy values
MSI (c) (AC:9C) [22:17:00:308]: Machine policy value 'Debug' is 0
MSI (c) (AC:9C) [22:17:00:308]: ******* RunEngine:
           ******* Product: Brackets.1.0.Extract.msi
           ******* Action: 
           ******* CommandLine: **********
MSI (c) (AC:9C) [22:17:00:398]: Machine policy value 'DisableUserInstalls' is 0
MSI (c) (AC:74) [22:17:00:812]: Font created.  Charset: Req=0, Ret=0, Font: Req=, Ret=Arial

MSI (c) (AC:74) [22:17:01:140]: Cloaking enabled.
MSI (c) (AC:74) [22:17:01:140]: Attempting to enable all disabled privileges before calling Install on Server
MSI (c) (AC:74) [22:17:01:220]: Font created.  Charset: Req=0, Ret=0, Font: Req=, Ret=Arial

MSI (c) (AC:9C) [22:17:01:230]: Note: 1: 1402 2: HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Policies\Explorer 3: 2 
MSI (c) (AC:9C) [22:17:01:257]: SOFTWARE RESTRICTION POLICY: Verifying package --> 'C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi' against software restriction policy
MSI (c) (AC:9C) [22:17:01:257]: SOFTWARE RESTRICTION POLICY: C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi has a digital signature
MSI (c) (AC:9C) [22:17:21:118]: SOFTWARE RESTRICTION POLICY: C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi is permitted to run at the 'unrestricted' authorization level.
MSI (c) (AC:9C) [22:17:21:121]: Cloaking enabled.
MSI (c) (AC:9C) [22:17:21:121]: Attempting to enable all disabled privileges before calling Install on Server
MSI (c) (AC:9C) [22:17:21:146]: End dialog not enabled
MSI (c) (AC:9C) [22:17:21:146]: Original package ==> C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi
MSI (c) (AC:9C) [22:17:21:146]: Package we're running from ==> C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi
MSI (c) (AC:9C) [22:17:21:211]: APPCOMPAT: Compatibility mode property overrides found.
MSI (c) (AC:9C) [22:17:21:243]: APPCOMPAT: looking for appcompat database entry with ProductCode '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}'.
MSI (c) (AC:9C) [22:17:21:253]: APPCOMPAT: no matching ProductCode found in database.
MSI (c) (AC:9C) [22:17:21:754]: MSCOREE not loaded loading copy from system32
MSI (c) (AC:9C) [22:17:21:796]: Machine policy value 'TransformsSecure' is 0
MSI (c) (AC:9C) [22:17:21:796]: User policy value 'TransformsAtSource' is 0
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'DisablePatch' is 0
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'AllowLockdownPatch' is 0
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'DisableMsi' is 0
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'AlwaysInstallElevated' is 0
MSI (c) (AC:9C) [22:17:21:831]: User policy value 'AlwaysInstallElevated' is 0
MSI (c) (AC:9C) [22:17:21:831]: Running product '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}' with user privileges: It's not assigned.
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'DisableLUAPatching' is 0
MSI (c) (AC:9C) [22:17:21:831]: Machine policy value 'DisableFlyWeightPatching' is 0
MSI (c) (AC:9C) [22:17:21:839]: APPCOMPAT: looking for appcompat database entry with ProductCode '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}'.
MSI (c) (AC:9C) [22:17:21:839]: APPCOMPAT: no matching ProductCode found in database.
MSI (c) (AC:9C) [22:17:21:841]: Transforms are not secure.
MSI (c) (AC:9C) [22:17:21:841]: PROPERTY CHANGE: Adding MsiLogFileLocation property. Its value is 'C:\Users\sondes\Desktop\old\brackets_log.txt'.
MSI (c) (AC:9C) [22:17:21:841]: Command Line: CURRENTDIRECTORY=C:\Users\sondes\Desktop\old CLIENTUILEVEL=0 CLIENTPROCESSID=3756 
MSI (c) (AC:9C) [22:17:21:841]: PROPERTY CHANGE: Adding PackageCode property. Its value is '{3FE94502-41DB-407D-A571-BB58029975F8}'.
MSI (c) (AC:9C) [22:17:21:841]: Product Code passed to Engine.Initialize:           ''
MSI (c) (AC:9C) [22:17:21:841]: Product Code from property table before transforms: '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}'
MSI (c) (AC:9C) [22:17:21:841]: Product Code from property table after transforms:  '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}'
MSI (c) (AC:9C) [22:17:21:841]: Product not registered: beginning first-time install
MSI (c) (AC:9C) [22:17:21:841]: PROPERTY CHANGE: Adding ProductState property. Its value is '-1'.
MSI (c) (AC:9C) [22:17:21:841]: Entering CMsiConfigurationManager::SetLastUsedSource.
MSI (c) (AC:9C) [22:17:21:841]: User policy value 'SearchOrder' is 'nmu'
MSI (c) (AC:9C) [22:17:21:841]: Adding new sources is allowed.
MSI (c) (AC:9C) [22:17:21:841]: PROPERTY CHANGE: Adding PackagecodeChanging property. Its value is '1'.
MSI (c) (AC:9C) [22:17:21:841]: Package name extracted from package path: 'Brackets.1.0.Extract.msi'
MSI (c) (AC:9C) [22:17:21:841]: Package to be registered: 'Brackets.1.0.Extract.msi'
MSI (c) (AC:9C) [22:17:21:844]: Note: 1: 2262 2: AdminProperties 3: -2147287038 
MSI (c) (AC:9C) [22:17:21:844]: Machine policy value 'AlwaysInstallElevated' is 0
MSI (c) (AC:9C) [22:17:21:844]: User policy value 'AlwaysInstallElevated' is 0
MSI (c) (AC:9C) [22:17:21:844]: Running product '{ABCC2682-885E-47D9-9EB8-2785C3FD82E3}' with user privileges: It's not assigned.
MSI (c) (AC:9C) [22:17:21:844]: PROPERTY CHANGE: Adding CURRENTDIRECTORY property. Its value is 'C:\Users\sondes\Desktop\old'.
MSI (c) (AC:9C) [22:17:21:844]: PROPERTY CHANGE: Adding CLIENTUILEVEL property. Its value is '0'.
MSI (c) (AC:9C) [22:17:21:844]: PROPERTY CHANGE: Adding CLIENTPROCESSID property. Its value is '3756'.
MSI (c) (AC:9C) [22:17:21:844]: PROPERTY CHANGE: Adding MsiSystemRebootPending property. Its value is '1'.
MSI (c) (AC:9C) [22:17:21:844]: TRANSFORMS property is now: 
MSI (c) (AC:9C) [22:17:21:844]: PROPERTY CHANGE: Adding VersionDatabase property. Its value is '300'.
MSI (c) (AC:9C) [22:17:21:846]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming
MSI (c) (AC:9C) [22:17:21:846]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\Favorites
MSI (c) (AC:9C) [22:17:21:846]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Network Shortcuts
MSI (c) (AC:9C) [22:17:21:846]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\Documents
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Printer Shortcuts
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Recent
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\SendTo
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Templates
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\ProgramData
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Local
MSI (c) (AC:9C) [22:17:21:849]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\Pictures
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Administrative Tools
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\ProgramData\Microsoft\Windows\Start Menu\Programs
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\ProgramData\Microsoft\Windows\Start Menu
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\Users\Public\Desktop
MSI (c) (AC:9C) [22:17:21:851]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Administrative Tools
MSI (c) (AC:9C) [22:17:21:854]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup
MSI (c) (AC:9C) [22:17:21:854]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
MSI (c) (AC:9C) [22:17:21:854]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Start Menu
MSI (c) (AC:9C) [22:17:21:854]: SHELL32::SHGetFolderPath returned: C:\Users\sondes\Desktop
MSI (c) (AC:9C) [22:17:21:871]: SHELL32::SHGetFolderPath returned: C:\ProgramData\Microsoft\Windows\Templates
MSI (c) (AC:9C) [22:17:21:871]: SHELL32::SHGetFolderPath returned: C:\Windows\Fonts
MSI (c) (AC:9C) [22:17:21:874]: Note: 1: 2898 2: MS Sans Serif 3: MS Sans Serif 4: 0 5: 16 
MSI (c) (AC:9C) [22:17:21:884]: MSI_LUA: Setting AdminUser property to 1 because this is the client or the user has already permitted elevation
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding AdminUser property. Its value is '1'.
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding Privileged property. Its value is '1'.
MSI (c) (AC:9C) [22:17:21:884]: Note: 1: 1402 2: HKEY_CURRENT_USER\Software\Microsoft\MS Setup (ACME)\User Info 3: 2 
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding USERNAME property. Its value is 'sondes'.
MSI (c) (AC:9C) [22:17:21:884]: Note: 1: 1402 2: HKEY_CURRENT_USER\Software\Microsoft\MS Setup (ACME)\User Info 3: 2 
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding DATABASE property. Its value is 'C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi'.
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding OriginalDatabase property. Its value is 'C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi'.
MSI (c) (AC:9C) [22:17:21:884]: Machine policy value 'MsiDisableEmbeddedUI' is 0
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding SourceDir property. Its value is 'C:\Users\sondes\Desktop\old\'.
MSI (c) (AC:9C) [22:17:21:884]: PROPERTY CHANGE: Adding SOURCEDIR property. Its value is 'C:\Users\sondes\Desktop\old\'.
MSI (c) (AC:74) [22:17:21:974]: PROPERTY CHANGE: Adding VersionHandler property. Its value is '5.00'.
=== Logging started: 15/03/36  22:17:22 ===
MSI (c) (AC:9C) [22:17:22:558]: Note: 1: 2205 2:  3: PatchPackage 
MSI (c) (AC:9C) [22:17:22:558]: Machine policy value 'DisableRollback' is 0
MSI (c) (AC:9C) [22:17:22:558]: User policy value 'DisableRollback' is 0
MSI (c) (AC:9C) [22:17:22:558]: PROPERTY CHANGE: Adding UILevel property. Its value is '5'.
MSI (c) (AC:9C) [22:17:22:560]: PROPERTY CHANGE: Adding ACTION property. Its value is 'INSTALL'.
MSI (c) (AC:9C) [22:17:22:560]: Doing action: INSTALL
Action 22:17:22: INSTALL. 
Action start 22:17:22: INSTALL.
MSI (c) (AC:9C) [22:17:22:590]: UI Sequence table 'InstallUISequence' is present and populated.
MSI (c) (AC:9C) [22:17:22:590]: Running UISequence
MSI (c) (AC:9C) [22:17:22:590]: PROPERTY CHANGE: Adding EXECUTEACTION property. Its value is 'INSTALL'.
MSI (c) (AC:9C) [22:17:22:590]: Doing action: FindRelatedProducts
Action 22:17:22: FindRelatedProducts. Searching for related applications
Action start 22:17:22: FindRelatedProducts.
FindRelatedProducts: Found application: {5594F5B4-B8BD-4165-AEEB-317C324F93BB}
MSI (c) (AC:9C) [22:17:22:598]: PROPERTY CHANGE: Adding WIX_DOWNGRADE_DETECTED property. Its value is '{5594F5B4-B8BD-4165-AEEB-317C324F93BB}'.
Action ended 22:17:22: FindRelatedProducts. Return value 1.
MSI (c) (AC:9C) [22:17:22:598]: Doing action: LaunchConditions
Action 22:17:22: LaunchConditions. Evaluating launch conditions
Action start 22:17:22: LaunchConditions.
Info 2898. For WixUI_Font_Normal textstyle, the system created a 'Tahoma' font, in 0 character set, of 13 pixels height.
A later version of Brackets is already installed.  Setup will now exit.
MSI (c) (AC:9C) [22:17:25:855]: Product: Brackets -- A later version of Brackets is already installed.  Setup will now exit.

Action ended 22:17:25: LaunchConditions. Return value 3.
MSI (c) (AC:9C) [22:17:25:858]: Doing action: FatalError
Action 22:17:25: FatalError. 
Action start 22:17:25: FatalError.
Info 2898. For WixUI_Font_Bigger textstyle, the system created a 'Tahoma' font, in 0 character set, of 19 pixels height.
Action 22:17:25: FatalError. Dialog created
MSI (c) (AC:80) [22:17:25:918]: Note: 1: 2731 2: 0 
Action ended 22:17:27: FatalError. Return value 2.
Action ended 22:17:27: INSTALL. Return value 3.
Property(C): UpgradeCode = {3B75C4E6-41C6-43E0-B0B2-92C7C7614AF1}
Property(C): ProgramMenuFolder = C:\ProgramData\Microsoft\Windows\Start Menu\Programs\
Property(C): WIXUI_INSTALLDIR = INSTALLDIR
Property(C): ALLUSERS = 1
Property(C): ProgramFilesFolder = C:\Program Files (x86)\
Property(C): SourceDir = C:\Users\sondes\Desktop\old\
Property(C): Manufacturer = brackets.io
Property(C): ProductCode = {ABCC2682-885E-47D9-9EB8-2785C3FD82E3}
Property(C): ProductLanguage = 1033
Property(C): ProductName = Brackets
Property(C): ProductVersion = 1.0
Property(C): ARPPRODUCTICON = appicon.ico
Property(C): DefaultUIFont = WixUI_Font_Normal
Property(C): WixUI_Mode = InstallDir
Property(C): ARPNOMODIFY = 1
Property(C): ARPNOREPAIR = 1
Property(C): ErrorDialog = ErrorDlg
Property(C): SecureCustomProperties = WIX_DOWNGRADE_DETECTED;WIX_UPGRADE_DETECTED
Property(C): WIX_DOWNGRADE_DETECTED = {5594F5B4-B8BD-4165-AEEB-317C324F93BB}
Property(C): MsiLogFileLocation = C:\Users\sondes\Desktop\old\brackets_log.txt
Property(C): PackageCode = {3FE94502-41DB-407D-A571-BB58029975F8}
Property(C): ProductState = -1
Property(C): PackagecodeChanging = 1
Property(C): CURRENTDIRECTORY = C:\Users\sondes\Desktop\old
Property(C): CLIENTUILEVEL = 0
Property(C): CLIENTPROCESSID = 3756
Property(C): VersionDatabase = 300
Property(C): MsiSystemRebootPending = 1
Property(C): VersionMsi = 5.00
Property(C): VersionNT = 601
Property(C): VersionNT64 = 601
Property(C): WindowsBuild = 7601
Property(C): ServicePackLevel = 1
Property(C): ServicePackLevelMinor = 0
Property(C): MsiNTProductType = 1
Property(C): MsiNTSuitePersonal = 1
Property(C): WindowsFolder = C:\Windows\
Property(C): WindowsVolume = C:\
Property(C): System64Folder = C:\Windows\system32\
Property(C): SystemFolder = C:\Windows\SysWOW64\
Property(C): RemoteAdminTS = 1
Property(C): TempFolder = C:\Users\sondes\AppData\Local\Temp\
Property(C): CommonFilesFolder = C:\Program Files (x86)\Common Files\
Property(C): ProgramFiles64Folder = C:\Program Files\
Property(C): CommonFiles64Folder = C:\Program Files\Common Files\
Property(C): AppDataFolder = C:\Users\sondes\AppData\Roaming\
Property(C): FavoritesFolder = C:\Users\sondes\Favorites\
Property(C): NetHoodFolder = C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Network Shortcuts\
Property(C): PersonalFolder = C:\Users\sondes\Documents\
Property(C): PrintHoodFolder = C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Printer Shortcuts\
Property(C): RecentFolder = C:\Users\sondes\AppData\Roaming\Microsoft\Windows\Recent\
Property(C): SendToFolder = C:\Users\sondes\AppData\Roaming\Microsoft\Windows\SendTo\
Property(C): TemplateFolder = C:\ProgramData\Microsoft\Windows\Templates\
Property(C): CommonAppDataFolder = C:\ProgramData\
Property(C): LocalAppDataFolder = C:\Users\sondes\AppData\Local\
Property(C): MyPicturesFolder = C:\Users\sondes\Pictures\
Property(C): AdminToolsFolder = C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Administrative Tools\
Property(C): StartupFolder = C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup\
Property(C): StartMenuFolder = C:\ProgramData\Microsoft\Windows\Start Menu\
Property(C): DesktopFolder = C:\Users\Public\Desktop\
Property(C): FontsFolder = C:\Windows\Fonts\
Property(C): GPTSupport = 1
Property(C): OLEAdvtSupport = 1
Property(C): ShellAdvtSupport = 1
Property(C): MsiAMD64 = 6
Property(C): Msix64 = 6
Property(C): Intel = 6
Property(C): PhysicalMemory = 4004
Property(C): VirtualMemory = 4561
Property(C): LogonUser = sondes
Property(C): UserSID = S-1-5-21-684317524-2884594227-2894737267-1000
Property(C): UserLanguageID = 7169
Property(C): ComputerName = SONDES-PC
Property(C): SystemLanguageID = 7169
Property(C): ScreenX = 1366
Property(C): ScreenY = 768
Property(C): CaptionHeight = 22
Property(C): BorderTop = 1
Property(C): BorderSide = 1
Property(C): TextHeight = 16
Property(C): TextInternalLeading = 3
Property(C): ColorBits = 32
Property(C): TTCSupport = 1
Property(C): Time = 22:17:27
Property(C): Date = 15/03/36
Property(C): MsiNetAssemblySupport = 4.0.30319.18408
Property(C): MsiWin32AssemblySupport = 6.1.7601.17514
Property(C): RedirectedDllSupport = 2
Property(C): AdminUser = 1
Property(C): Privileged = 1
Property(C): USERNAME = sondes
Property(C): DATABASE = C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi
Property(C): OriginalDatabase = C:\Users\sondes\Desktop\old\Brackets.1.0.Extract.msi
Property(C): SOURCEDIR = C:\Users\sondes\Desktop\old\
Property(C): VersionHandler = 5.00
Property(C): UILevel = 5
Property(C): ACTION = INSTALL
Property(C): EXECUTEACTION = INSTALL
=== Logging stopped: 15/03/36  22:17:27 ===
MSI (c) (AC:9C) [22:17:27:041]: Note: 1: 1708 
MSI (c) (AC:9C) [22:17:27:043]: Product: Brackets -- Installation failed.

MSI (c) (AC:9C) [22:17:27:043]: Windows Installer installed the product. Product Name: Brackets. Product Version: 1.0. Product Language: 1033. Manufacturer: brackets.io. Installation success or error status: 1603.

MSI (c) (AC:9C) [22:17:27:111]: Grabbed execution mutex.
MSI (c) (AC:9C) [22:17:27:111]: Cleaning up uninstalled install packages, if any exist
MSI (c) (AC:9C) [22:17:27:116]: MainEngineThread is returning 1603
=== Verbose logging stopped: 15/03/36  22:17:27 ===

```

I hope you find out something from this long piece of code...
Good luck!

**Comment by mobesa** [comment]
Wow!
I have found another issue (#8224) and it mentioned the same problem I have: I have StarUML installed on my device... it seems that StarUML is conflicting with brackets. Might as well uninstall it and start working with Brackets!!!
Well, I would love to close this *ticket*, but I am not sure if I should; maybe there are other users out there who have StarUML installed and who need Brackets alongside it... So, I think we have to find a well-rounded solution here folks; to remove the conflict between these apps once and for all, somehow...
Cheers!

**Comment by redmunds** [comment]
@mobesa So, you saying that if you uninstall StarUML then you can install Brackets?

It looks like there could be an app GUID conflict:

```
FindRelatedProducts: Found application: {5594F5B4-B8BD-4165-AEEB-317C324F93BB}
```

Wow! The odds of that are pretty low. We could change the Brackets app GUID, but that would mess up future updates. Since we just release Brackets 1.0, this is probably the best time to make such a change.

**Comment by thany** [comment]
This wouldn't fix being unable to downgrade Bracket 1.1 to 1.0.

Such check does not need to exist in the first place.

**Comment by redmunds** [comment]
@thany This would allow you to install 1.0 side by side with 1.2 and greater, so there would be no *downgrade*.

**Comment by mobesa** [comment]
@redmunds : That's right! I uninstalled StarUML and then installed Brackets 1.1 + Extract and all went smoothly...

**Comment by thany** [comment]
@redmunds Ehh, no. If I let Brackets install itself into the same directory, it should overwrite whatever is already there. For all intents and purposes, that's a downgrade (or an upgrade for that matter).

**Comment by peterflynn** [comment]
It's no coincidence: looking at the website (http://staruml.io/), it's very clear that StarUML is heavily reusing code from the Brackets project.  So they presumably just copied our installer code without changing the product GUID, which is a big no-no.  We should contact them and ask them to fix it on their end.

We also may want to consider removing our GUID from the source code, so that it's not possible to build an installer without generating your own GUID...

**Comment by peterflynn** [comment]
I've pinged them, so hopefully they'll correct their installers for the future.  It won't help anyone who's _already_ installed StarUML, but once fixed it will at least prevent future installs from creating this problem.

**Comment by peterflynn** [comment]
Also, to clarify two things from earlier in the thread:

* `msiexec /a` is _not_ installing as an admin (`/a` doesn't mean "admin," it means something akin to "don't install, unzip instead" ...and it's tricky to use since it may fail silently if you don't provide other info like a valid target path). To install as an admin, [follow these steps](https://github.com/adobe/brackets/wiki/Troubleshooting#windows-error-installation-directory-must-be-on-a-local-drive) (which use `/i`, not `/a`).
* As already noticed here, Windows installers (MSI files) operate off of GUIDs.  So searching the Registry for "Brackets" won't help with cases like this since MSIs don't go off of the app name. They only care about the GUID, which can be found via installer logs like the one above.

**Comment by mobesa** [comment]
@peterflynn thanks man! now I can close this issue in peace...

**Comment by redmunds** [comment]
Re-opening. Let's keep this open until we verify that issue has been resolved.

**Comment by peterflynn** [comment]
Marking tracking since we're waiting on StarUML to release a fix on their end

**Comment by peterflynn** [comment]
We should also consider removing our GUID from the source code too though, so it's harder for developers to make this mistake

**Comment by Rheam** [comment]
Hi, I have the same problem. However, I have solved the problem a little easier. My recommendation for all who have ended up on Google search here (like me). There is a program that is called "CCleaner" (it's freeware & portable). Goes to "Tools" -> Uninstall Tab. Here you will find the Brackets program which unfortunately Windows does not display. Namely an old or faulty installation of Brackets. Simply remove Brackets. CCleaner also cleans up registry. If you like.

**Comment by mobesa** [comment]
@Rheam thanks for the suggestion! Glad to hear that you've solved it by an easier method.
